robot/Cumulus/tests/browser/gift_entry/template_builder.robot

```diff
@@ -40,11 +44,13 @@ Create Clone and Delete Template
     Click Gift Entry Button                   Save & Close
     Wait Until Page Contains                  This name has been used by another template. Please enter a unique name.
     Enter Value In Field                      Template Name=${new_template}
+    Click Gift Entry Button                   Next: Form Fields
+    Click Gift Entry Button                   Next: Batch Settings
```

Comment:
why add these 2 steps when we are not doing anything on those tabs?

Reply:
This was the main cause of the failure - the form throws an error if you don't click to the Form Fields tab and skip right to saving. I had it mimic creating the previous template to ensure parity.